roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -100,13 +100,22 @@
     files:
     - "{{ tempdir }}/logging-elasticsearch-view-role.yaml"
     delete_after: true
-
 # configmap
+- fail:
```

Comment:
can we change this from a `fail` task to an `assert` task? we should eventually convert all the cases of it in the role... this would make one less to convert later